### DISCUSSION TITLE
Is it possible to have dynamic minecraft (item, block, etc.) names? Give user the ability to rename the item

### DISCUSSION THREAD

**Discussion by AgentKosticka** [FabricMC/fabric-api#2340] [Mod Dev Support] [answered]
I know how to change the item's name, but it will allways look like net.blabla.blabla.uranium or net.blabla.blabla.mordirium or else. Is it possible to make 1 fit all size thing in the eu_us.json file (keep in mind that the name isnt limited(it's not like the user picks from 2 or 3 options))

**Comment by warjort** [comment, marked as answer]
Why doesn't an anvil do what you want? Or if you want to emulate it, ````ItemStack.setCustomName()````.
